### DISCUSSION TITLE
Data sanitisation

### DISCUSSION THREAD

**Discussion by eincandela** [PostgREST/postgrest#1962] [Q&A] [answered]
Hello, i'm a newbie on this great project and i cannot find how to, if possible, validate/filter/sanitize incoming inputs (like email, range of values for numbers, etc) like we do in a classic rest api.
Can  you lead me on some good practice ?
Thanks :)

**Comment by steve-chavez** [comment, marked as answer]
Hey @eincandela,

Validation is done through check constraints, check this [tutorial](https://www.postgresqltutorial.com/postgresql-check-constraint/) for how to use them.
